src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -2012,6 +2013,7 @@ void grpc_chttp2_cancel_stream(grpc_chttp2_transport* t, grpc_chttp2_stream* s,
   if (!t->is_client && !s->sent_trailing_metadata &&
       grpc_error_has_clear_grpc_status(due_to_error)) {
     close_from_api(t, s, due_to_error);
+    GRPC_CLOSURE_LIST_SCHED(&s->run_after_write);
```

Comment:
shouldn't be needed right? remove_stream already has it

Reply:
Good observation, I've removed it.